nodejs/package.json

```diff
@@ -0,0 +1,21 @@
+{
+  "name": "cs50-id",
+  "version": "1.0.0",
+  "description": "Sample code for using CS50 ID to authenticate users via HarvardKey, Princeton CAS, or Yale CAS.",
+  "main": "app.js",
+  "dependencies": {
+    "cookie-parser": "^1.4.5",
+    "ejs": "^3.1.6",
+    "express": "^4.17.1",
+    "express-session": "^1.17.2",
+    "node-fetch": "^2.6.1",
+    "passport": "^0.4.1",
+    "passport-oauth2": "^1.6.0"
+  },
+  "devDependencies": {},
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1"
+  },
```

Comment:
Let's omit `scripts` as unneeded? And `devDependencies` and any other keys that `npm` doesn't require?